src/plugins/gui/fiatPlugin.tsx

```diff
@@ -78,21 +85,51 @@ export const executePlugin = async (params: {
     },
     popScene: async () => {
       navigation.pop()
+    },
+    enterFieldsForm: async (params: FiatPluginSepaFormParams) => {
```

Comment:
The type `FiatPluginSepaFormParams` and the gui method names don't match. The params imply Sepa, the name implies a generic FieldsForm. Per my previous comment, this should be broken up into two different scenes.